schemas/bedrock-kms-http.js

```diff
@@ -0,0 +1,139 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const controller = {
+  title: 'controller',
+  type: 'string'
+};
+
+const postKeystore = {
+  title: 'postKeystore',
+  type: 'object',
+  additionalProperties: true,
```

Comment:
is it not possible do lock this down at this time?  If so, we should have a task related to nailing it down.

Reply:
After further testing I have added the `referenceId` property and changed this to `false`.